### PR TITLE
Add Deno.version and Deno.pid examples

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: data/deno-version.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @title Getting the Deno version
+ * @difficulty beginner
+ * @tags cli
+ *
+ * How to examine the version of Deno being used.
+ */
+
+// File: ./main.ts
```
Comment (lucacasonato) [inline data/deno-version.ts:9]: Not needed.

File: data/deno-version.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @title Getting the Deno version
+ * @difficulty beginner
+ * @tags cli
+ *
```
Comment (lucacasonato) [inline data/deno-version.ts:5]:
```suggestion
 * @run <url>
 * @resource {https://doc.deno.land/deno/stable/~/Deno.version} Doc: Deno.version
 *
```

File: data/pid.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ * @tags cli
+ * @run <url>
+ * @resource {https://deno.land/#installation} Deno: Installation
+ * @resource {https://deno.land/manual@v1.17.2/getting_started/setup_your_environment} Manual: Set up your environment
+ */
+
+// The process id is easily accessible in the `Deno` namespace
+console.log(Deno.pid);
+
+// As is the parent process id.
```
Comment (lucacasonato) [inline data/pid.ts:13]:
```
// The parent process ID is available in the Deno namespace too.
```

File: data/pid.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @title Process Information
+ * @difficulty beginner
+ * @tags cli
+ * @run <url>
+ * @resource {https://deno.land/#installation} Deno: Installation
+ * @resource {https://deno.land/manual@v1.17.2/getting_started/setup_your_environment} Manual: Set up your environment
+ */
+
+// The process id is easily accessible in the `Deno` namespace
```
Comment (lucacasonato) [inline data/pid.ts:10]: Don't assume things are easy for people :D

Maybe 
```
The current processes' process ID is available in the `Deno.pid` variable.
```

File: data/pid.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @title Process Information
```
Comment (lucacasonato) [inline data/pid.ts:2]:
```suggestion
 * @title Process Information
```

File: data/pid.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ * @difficulty beginner
+ * @tags cli
+ * @run <url>
+ * @resource {https://deno.land/#installation} Deno: Installation
+ * @resource {https://deno.land/manual@v1.17.2/getting_started/setup_your_environment} Manual: Set up your environment
```
Comment (lucacasonato) [inline data/pid.ts:7]: Not relevant.

File: data/pid.ts (hunk 5 above)
Comment (ry) [reply data/pid.ts:2]: what has changed here..?

Comment (lucacasonato) [reply data/pid.ts:2]: Nothing, my bad.

**Pull request merged**